src/python/pants/backend/awslambda/python/rules_test.py

```diff
@@ -45,7 +45,11 @@ def create_python_awslambda(rule_runner: RuleRunner, addr: str) -> Tuple[str, by
     )
     built_asset = rule_runner.request(BuiltPackage, [PythonAwsLambdaFieldSet.create(target)])
     assert created_awslambda.digest == built_asset.digest
-    assert created_awslambda.zip_file_relpath == built_asset.relpath
+    assert created_awslambda.zip_file_relpath == built_asset.artifacts[0].relpath
```

Comment:
As below, if we expect there to be only a single artifact, we could make that explicit with a `.single_artifact` method or `@property`.